src/App.tsx

```diff
@@ -65,6 +66,8 @@ function App() {
   const { isAuthenticated, isLoading, getIdTokenClaims } = useAuth0();
 
   const [notifications, setNotifications] = useState<Notification[]>([]);
+  const [isFormVisible, setIsFormVisible] = useState(true);
```

Comment:
```suggestion
  const [isQueryFormOpen, setIsQueryFormOpen] = useState(true);
```
again, mainly for clarity! This state variable doesn't control visibility directly, it controls open-close status for the form.